outcomes/firefox_desktop/spotlight_engagement.toml

```
@@ -1,24 +1,41 @@
-friendly_name = "Spotlight Engagement - not experiment specific"
+### outcome properties
+friendly_name = "Spotlight Engagement - not experiment specific"  # TODO: is the not experiment specific still accurate?
 description = "Usage and engagement metrics for the Spotlight feature - for ALL engagement with Spotlight feature, not experiment specific "
 
+
+### parameters definition
+[parameters]
+
+[parameters.message_id]
+friendly_name = "Message ID associated with the experiment"
+description = "ID of the spotlight message associated with this experiment"
+default = "'AND DIV(7, 0)' -- Failed to specify a value for message_id"
+
+[parameters.branch]
+friendly_name = "Branch name associated with the experiment"
+description = "Branch name associated with this experiment's message id"
+default = "'AND DIV(11, 0)' -- Failed to specify a value for branch_id"
```

So for spotlight there could be multiple different branches that each show a different message (with a different message ID). To indicate this, we'll need to set `distinct_by_branch = true`.

```suggestion
distinct_by_branch = true
```

---

this can be overwritten with external config